_posts/2019-02-02-digitalocean-one-click.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Announcing DigitalOcean's OpenFaaS One-click"
+description: Earlier this week saw the release of the first OpenFaaS One-click application on DigitalOcean.  Richard Gee guides you through just how simple it is to create your first OpenFaaS function on DigitalOcean.
+date: 2019-02-02
+image: /images/digitalocean-one-click/balance-macro-ocean.jpg
+categories:
+  - developer-experience
+  - swarm
+  - python
+author_staff_member: richard
+dark_background: false
+---
+
+I've written previously about how you can be [up and running in 5 minutes](/blog/deploy-digitalocean-ansible.md) with OpenFaaS on DigitalOcean.  With the news this week that DigitalOcean are now offering an OpenFaaS One-click application this time shortens further and increases the number of cost effective ways in which to run OpenFaaS on DigitalOcean:
+
+* [Vanilla Droplets running Swarm or Kubernetes](/blog/deploy-digitalocean-ansible.md)
+* Using the [DigitalOcean Kubernetes Service (DOKS)](https://blog.alexellis.io/digitalocean-kubernetes-engine/)
+* One-click application creating a single Ubuntu node Swarm-based instance
+
+In this post we'll guide you through creating your first OpenFaaS function using the One-Click application.
```

Comment:
What do they call it in their UI, is it "One-Click" or "Droplet Image" or etc?

Reply:
One-click